apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -439,6 +438,35 @@ export class NotificationService {
     )
   }
 
+  private async defenderCourtDateNotificationSent(theCase: Case) {
+    let exists = false
+
+    try {
+      const notifications = await this.notificationModel.findAll({
+        where: {
+          caseId: theCase.id,
+          type: NotificationType.COURT_DATE,
+        },
+      })
+
+      exists = notifications.some(async (notification) => {
+        JSON.parse(notification.recipients ?? '[]').some(
```

Comment:
Looks like there is a missing return here
```suggestion
        return JSON.parse(notification.recipients ?? '[]').some(
```

Reply:
Good catch. However, the code works as is, probably because javascript returns the result of the last expression in the block. Will fix.